File: 1semana/Livro.java
public class Livro {
    // Atributos da classe
    public String titulo;
    public String autor;
    public int ano;
    // Método para exibir
    public void exibir() {
        System.out.println("Titulo: " 
        + titulo + "\nAutor:" + autor 
        + "\nAno de Lancamento:" + ano);
    }
    public static void main(String[] args) {
        // Instância da classe
        Livro l = new Livro();

        // Atribuição de valores
        l.titulo = "LPOO";
        l.autor = "Tanembaum";
        l.ano = 2024;

        // Chamada do método para exibir
        l.exibir();
    }
}

File: 1semana/Livro.py
class Livro:
    def __init__(self, titulo, autor, ano):
        self.titulo = titulo
        self.autor = autor
        self.ano = ano

    def exibir(self):
        print(f"Titulo: {self.titulo} \nAutor: {self.autor}\
            \nLancamento: {self.ano}!")
# Criando uma instância da classe
p = Livro("LPOO", "Tanembaum", 2024)
# Chamando o método exibir
p.exibir()